# A representation of pose in free space, composed of position (x, y, z) and orientation (only yaw) in degrees
#Point pos
#Rotation rot
float64 x
float64 y
float64 z
float64 yaw
